### ISSUE TITLE
Frame rate stuttering when flying over Bastion

### ISSUE THREAD

**Issue by tflo** [Breeni/BtWTodo#3]
This is very likely related to my other issue https://github.com/Breeni/BtWTodo/issues/1.

I'm rarely in Bastion, so I don't know when this started to happen, but the symptoms are very similar to the ones of the issue linked above, though the stuttering is less frequent and less intense; it is (at least to me) only noticeable while flying.

Here a screeny of the event tracer:

![WoWScrnShot_092421_154754](https://user-images.githubusercontent.com/1410282/134686768-646eba80-fbef-441e-ba3c-7484b13e0c2f.jpg)

It _seems_ to be again the QUEST_DATA_LOAD_RESULT event.

Stuttering goes away if I unload the add-on.

Sorry for the somewhat sloppy issue report, let me know if you need more info. (For example, I haven't thoroughly checked if it really only ever happens in Bastion, or if it also happens in other zones when certain conditions are met.)

**Comment by Breeni** [comment]
Most of those QUEST_DATA_LOAD_RESULT events are from world quests, although BtWTodo does listen to those events, it is only for quests it is tracking and not those world quests and even then it should only react once.

**Comment by tflo** [comment]
It happens also in other zones, but in Korthia for example, where we can’t fly and are fighting a lot, it doesn’t stay out as it does in a “calm” zone wehere we just fly through.

Though it is not game-disrupting like the old issue with the modifier keys, it’s a somewhat ugly experience.

Let me know if you can’t reproduce it, then it might be an interference with some other add-on or WA, and I will try to isolate it.

**Comment by Breeni** [comment]
Any extra info you can give would be awesome as I am having issues reproducing the issue.

**Comment by tflo** [comment]
OK. Good news: it is not BtWTodo.

My initial testing was indeed sloppy, too sloppy. The responsible is the [QuestClicks](https://www.curseforge.com/wow/addons/questclicks) add-on, more precisely the zone-change-related `RegisterEvent`s of that add-on.

Not sure why I thought it was your add-on. Probably I was biased because of the other issue, which had very similar symptoms, and when I unloaded your add-on for testing I apparently wasn't aware that I was running an add-on set where the other one was not loaded.

Well, lesson learned: never ever do sloppy testing when going to post an issue.

I’m really sorry for the noise I produced!
